src/main/java/com/st/lms/models/Book.java

```diff
@@ -0,0 +1,76 @@
+package com.st.lms.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
+@Entity
+@Table(name = "tbl_book")
+public class Book {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private int bookId;
+	
+	@Column(name = "title")
+	private String title;
+	
+	@Column(name = "authId")
+	private int authorId;

```

Comment:
As I've said in previous reviews, model objects should include *object references* to other objects, *not* the ID numbers to look up those objects in the database!